xbmc/addons/GUIWindowAddonBrowser.cpp

```diff
@@ -184,14 +184,18 @@ bool CGUIWindowAddonBrowser::OnClick(int iItem, const std::string &player)
       g_mediaManager.GetNetworkLocations(shares);
       std::string path;
       if (CGUIDialogFileBrowser::ShowAndGetFile(shares, "*.zip", g_localizeStrings.Get(24041), path))
+      {
         CAddonInstaller::GetInstance().InstallFromZip(path);
+        Refresh();
```

Comment:
the name of the method implies that an addon gets installed and should appear in the list. is this not the case? if vecItems is changed, Refresh needs to,be called